dist/nginx.conf

```diff
@@ -52,21 +52,21 @@ server {
   gzip_http_version 1.1;
   gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript image/svg+xml image/x-icon;
 
-  add_header Strict-Transport-Security "max-age=31536000" always;
+  # add_header Strict-Transport-Security "max-age=31536000" always;
 
   location / {
     try_files $uri @proxy;
   }
 
   location ~ ^/(emoji|packs|system/accounts/avatars|system/media_attachments/files) {
     add_header Cache-Control "public, max-age=31536000, immutable";
-    add_header Strict-Transport-Security "max-age=31536000" always;
+    # add_header Strict-Transport-Security "max-age=31536000" always;
```

Comment:
This should stay. This isn't served by Rails but by nginx itself.

Reply:
Thanks. The reason this was commented was because I needed to test those paths before I delete them.

Reply:
Can be replaced with:
```
    add_header Strict-Transport-Security "max-age=63072000; includeSubDomains";
```

Reply:
It seems I forgot to specify `always;`.

Reply:
Checked, there is no need to add 'always';